README.local-opsworks.md

```diff
@@ -0,0 +1,96 @@
+# Local opsworks development
+
+This feature allows you to spin up clusters that match our deployed opsworks
+clusters very closely in a local vagrant environment.
+
+## Requirements
+
+* virtualbox 5.0.x
+* vagrant 1.8.x
+* [vagrant-hosts plugin](https://github.com/oscar-stack/vagrant-hosts)
+* mh-opsworks-recipes and matterhorn-dce-fork must be checked out as sibling
+  directories of this repository.
+
+## Getting started
+
+* Checkout mh-opsworks and run `./bin/setup`.
+* Install the required versions of vagrant and virtual box
+* Install the vagrant-hosts plugin:
+
+        vagrant plugin install vagrant-hosts --plugin-version 2.6.2
+
+* Ensure your repository directory structure looks like this:
+
+        [parent dir]
+          - mh-opsworks
+          - mh-opsworks-recipes
+          - matterhorn-dce-fork
+
+* Start your environment! This takes about 20 to 25 minutes on my beefy machine
```

Comment:
Ok - resuming and suspending a previously provisioned cluster takes around 50 seconds each way now that I've parallelized it.